R/helpers.R

```diff
@@ -364,6 +364,30 @@ cells_summary <- function(groups = NULL,
   cells
 }
 
+#' @rdname location_cells
+#' @import rlang
+#' @export
+cells_grand_summary <- function(columns = NULL,
```

Comment:
I assume valid options for `columns` are the usual suspects (logical, numeric, character, `NULL`; all `!!` aware), but what about for `rows`? Summary names, I assume? Logical/numeric too?

Reply:
This will take some doing... row names for summary rows (be it groupwise summaries or grand summaries) are available if the user supplies them. If they are not supplied, then during render time (in `create_summary_dfs()` <`utils_render_common.R`>) they are created from quoted function names (e.g., `"mean"`) or from RHS formulas (e.g., `~mean(., na.rm = TRUE)`).

A good option I think is to have the creation of names (those not explicitly supplied) performed right in `summary_rows()`. Then we could have those names stored in the `summary` attribute for both resolving positions and for rendering those labels.

Reply:
The `rows` argument is now as powerful as `columns`.